libr/fs/p/fs_io.c

```diff
@@ -27,26 +32,30 @@ static RFSFile* fs_io_open(RFSRoot *root, const char *path) {
 static bool fs_io_read(RFSFile *file, ut64 addr, int len) {
 	RFSRoot *root = file->root;
 	// char *cmd = r_str_newf ("mg %s %"PFMT64x" %d", file->path, addr, len);
-	char *cmd = r_str_newf ("mg %s", file->path);
+	char *cmd = r_str_newf ("mg %s", file->name);
```

Comment:
uhm, not sure about this change

Reply:
yeah, me neither. the fact is in my tests here `file->path` is always NULL. i'm looking for the root cause of that.

Reply:
so the `file->path` field seems pretty unused

Reply:
oh ok